src/v2/Components/Authentication/Types.ts

```diff
@@ -66,7 +66,10 @@ export interface ModalOptions {
    * }
    */
   afterSignUpAction?: AfterSignUpAction
-
+  /*
+   * Whether or not the user is using the form via gravity's oauth flow.
+   */
+  api_login?: boolean
```

Comment:
"API" doesn't have much to do with this (it just happens that the API host is responsible for oauth). Could this be renamed `oauthLogin` or something that makes its intent obvious like `resetLogin`?